openwisp_radius/api/serializers.py

```diff
@@ -102,11 +103,23 @@ class Meta:
         fields = '__all__'
 
 
+class RadiusOrganizationField(serializers.SlugRelatedField):
+    def get_queryset(self):
+        queryset = Organization.objects.all()
+        request = self.context.get('request', None)
+        if not request.user.is_superuser:
+            queryset = queryset.filter(pk__in=request.user.organizations_dict.keys())
+        return queryset
+
+
 class RadiusBatchSerializer(serializers.ModelSerializer):
+    organization = serializers.PrimaryKeyRelatedField(read_only=True)
+    organization_slug = RadiusOrganizationField(
+        required=True, label='organization', slug_field='slug', write_only=True
+    )
```

Comment:
`organization_slug` is used for input and `organization` is used for returning `pk` of organization.